# Goal
string structure_name
string structure_type
---
# Result
bool success
---
# Feedback
string status
